### PR TITLE
bottom navigation bar

### PR DESCRIPTION
**Pull request by DGojka**
Adding bottom navigation bar for future features, i.e. **`Trainings`** and **`Meta Trainings`**.

**`Task:`** https://trello.com/c/E7S327vo/38-add-navigation-bottom-bar

### CODE REVIEW

File: app/src/main/java/com/example/stretchy/activity/MainActivity.kt [hunk 1]
```diff
@@ -65,6 +79,61 @@ class MainActivity : ComponentActivity() {
...
     }
+    @Composable
+    fun BottomNavigationBar() {
+        val navController = rememberNavController()
+
+        val screens = listOf(
+            BottomNavScreen("Screen 1", Icons.Default.Home, "screen1"),
+            BottomNavScreen("Screen 2", Icons.Default.Search, "screen2"),
+            BottomNavScreen("Screen 3", Icons.Default.Person, "screen3"),
+        )
```
Comment (PPetka) [inline app/src/main/java/com/example/stretchy/activity/MainActivity.kt:90]: Screens should have their name set, those should be in strings. I propose 
`Stretching`
`Training`
`Meta Training`

File: app/src/main/java/com/example/stretchy/activity/MainActivity.kt [hunk 2]
```diff
@@ -65,6 +79,61 @@ class MainActivity : ComponentActivity() {
...
+                    }
+                }
+            }
+        ) { innerPadding ->
+            NavHost(
+                navController = navController,
+                startDestination = "screen1",
+                modifier = Modifier.padding(innerPadding)
+            ) {
+                composable("screen1") {
```
Comment (PPetka) [inline app/src/main/java/com/example/stretchy/activity/MainActivity.kt:116]: applies also here

**Review by PPetka — CHANGES_REQUESTED** [review]
I know its a draft, however ☝️ few comments were added.

Additionally
```
<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="20dp" android:height="20dp" android:viewportWidth="20" android:viewportHeight="20" android:tint="?attr/colorControlNormal">
<path android:fillColor="@android:color/white" android:pathData="M12.92,7.68l-0.5,-0.87c-0.03,-0.06 -0.1,-0.08 -0.15,-0.06l-0.62,0.25c-0.13,-0.1 -0.27,-0.18 -0.42,-0.24l-0.09,-0.66C11.12,6.04 11.06,6 11,6h-1C9.94,6 9.89,6.04 9.88,6.11L9.78,6.77C9.63,6.83 9.49,6.92 9.36,7.01L8.74,6.76c-0.06,-0.02 -0.12,0 -0.15,0.06l-0.5,0.87C8.06,7.74 8.07,7.81 8.12,7.84l0.53,0.41C8.63,8.33 8.62,8.42 8.62,8.5c0,0.08 0.01,0.17 0.02,0.24L8.12,9.16C8.07,9.19 8.06,9.26 8.08,9.32l0.5,0.87c0.03,0.06 0.1,0.08 0.15,0.06l0.62,-0.25c0.13,0.1 0.27,0.18 0.42,0.24l0.09,0.66C9.89,10.96 9.94,11 10,11h1c0.06,0 0.12,-0.04 0.12,-0.11l0.09,-0.66c0.15,-0.06 0.29,-0.15 0.42,-0.24l0.62,0.25c0.06,0.02 0.12,0 0.15,-0.06l0.5,-0.87c0.03,-0.06 0.02,-0.12 -0.03,-0.16l-0.52,-0.41c0.01,-0.08 0.02,-0.16 0.02,-0.24c0,-0.08 -0.01,-0.17 -0.02,-0.24l0.53,-0.41C12.93,7.81 12.94,7.74 12.92,7.68zM10.5,9.5c-0.55,0 -1,-0.45 -1,-1s0.45,-1 1,-1s1,0.45 1,1S11.05,9.5 10.5,9.5z"/>
<path android:fillColor="@android:color/white" android:pathData="M10.5,4C8.19,4 6.3,5.75 6.05,8l-1.53,2.22C4.3,10.55 4.53,11 4.93,11H6v2c0,0.55 0.45,1 1,1h1v2h5v-3.76c1.21,-0.81 2,-2.18 2,-3.74C15,6.01 12.99,4 10.5,4zM12,11.65V15H9v-2H7v-3H5.89l1.13,-1.64C7.09,6.5 8.61,5 10.5,5C12.43,5 14,6.57 14,8.5C14,9.89 13.18,11.09 12,11.65z"/>
</vector>


```

**Review by PPetka — APPROVED** [review]

**Comment by DGojka** [comment]
Bottom navigation bar

**Pull request merged**